projects/ad-acevsecrdset-sl/src/self_test/self_test.c

```diff
@@ -224,7 +235,7 @@ int self_test_rcd(struct stout *stout)
 	if (ret)
 		return ret;
 
-	no_os_mdelay(40);
```

Comment:
is this a related change to what it is described in the commit body/title? (also adding a comment on the meaning of this delay)

Reply:
We created a separate commit.